robot_calibration/include/robot_calibration/capture/depth_camera.h

```diff
@@ -37,21 +37,22 @@ class DepthCameraInfoManager
 
   bool init(ros::NodeHandle& n)
   {
-    camera_info_subscriber_ = n.subscribe("/head_camera/depth/camera_info",
+    std::string topic_name;
+    n.param<std::string>("camerainfo", topic_name, "/camera_info");
```

Comment:
this will require changes to the fetch_calibration package -- which is not ideal -- we should keep the same default -- we should also name this parameter "camera_info" since that is more consistent with the actual topic names